vm/src/stdlib/time.rs

```diff
@@ -704,6 +710,136 @@ mod windows {
         }
     }
 
+    fn win_perf_counter_frequency(vm: &VirtualMachine) -> PyResult<i64> {
+        let freq = unsafe {
+            let mut frequency = std::mem::MaybeUninit::uninit();
+            if QueryPerformanceFrequency(frequency.as_mut_ptr()) == 0 {
+                // TODO: modify error message more properly
+                return Err(vm.new_os_error("Failed to get performance frequency".to_owned()));
+            }
+            frequency.assume_init()
+        };
+
+        let frequency = unsafe { freq.QuadPart() };
+        if *frequency < 1 {
+            return Err(vm.new_runtime_error("invalid QueryPerformanceFrequency".to_owned()));
+        }
+
+        // TODO: is this overflow check essential?
+        if *frequency > i64::MAX / SEC_TO_NS {
+            return Err(vm.new_overflow_error("QueryPerformanceFrequency is too large".to_owned()));
+        }
```

Comment:
I don't think so (the comment in the equivalent check in CPython notes that this shouldn't happen with 64bit values) but I guess its best if we still keep it.